rsts/deployment/cluster_config/auth_setup.rst

```diff
@@ -85,8 +85,10 @@ Flyte supports connecting with external OIdC providers. Here are some examples f
 
     .. note::
 
-      Make sure to create an OAuth2 Client Credential. The `client_id` and `client_secret` will be needed in the following
+      Make sure to:   
+      * Create an OAuth2 Client Credential. The `client_id` and `client_secret` will be needed in the following
       steps.
+      * Configure `https://localhost:30081/callback` as a `redirect URI <https://developers.google.com/identity/openid-connect/openid-connect#setredirecturi>__.`
```

Comment:
This is only for the sandbox deployment, right? If yes, maybe explain this.

Reply:
Thank you @fg91 and @wild-endeavor. Good catch :)